### PR TITLE
feat: add loading spinner and make it work

### PR DESCRIPTION
**Pull request by T0ffik**

### CODE REVIEW

File: src/components/Spinner/_Spinner.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-index: 10;
+    height: 100%;
+    width: 100%;
+    background-color: $transparent-black;
+    position: fixed;
+    .spinner {
+        border: 16px solid #f3f3f3; /* Light grey */
```
Comment (Veryyapeee) [inline src/components/Spinner/_Spinner.scss]: these colors should be variables, also blue doesn't match our theme, it should be yellow like or gray

File: src/screens/Base.js [hunk 2]
```diff
@@ -24,6 +26,7 @@ export const BaseScreen = ({ renderOn }) => {
 
     Header({ renderOn: '#app-header' });
     Footer({ renderOn: '#app-footer' });
+    Spinner({ renderOn: '#spinner-place', loading: true });
```
Comment (Veryyapeee) [inline src/screens/Base.js]: loading: true? bruh, it's constantly spinning, it should only display while data from api is loading ;-;

**Review by Veryyapeee — DISMISSED** [review]

**Comment by Veryyapeee** [comment]
It's not quite responsive, it's too big on mobile view port

File: src/components/Spinner/_Spinner.scss (hunk 1 above)
Comment (T0ffik) [reply src/components/Spinner/_Spinner.scss]: Done!

File: src/screens/Base.js (hunk 2 above)
Comment (T0ffik) [reply src/screens/Base.js]: Resolved

**Review by Veryyapeee — APPROVED** [review]

**Review by Mariusz-web — APPROVED** [review]

**Pull request merged**